content/docs/reference/pkg/alicloud/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "alicloud"
+block_external_search_index: true
+menu:
+    reference:
+        parent: API Reference
+---
+
+<!-- WARNING: this file was generated by Pulumi Docs Generator. -->
+<!-- Do not edit by hand unless you're certain you know what you are doing! -->
+
+A Pulumi package for creating and managing AliCloud resources.
+
+> This provider is a derived work of the [Terraform Provider](https://github.com/terraform-providers/terraform-provider-alicloud)
+> distributed under [MPL 2.0](https://www.mozilla.org/en-US/MPL/2.0/). If you encounter a bug or missing feature,
+> first check the [`pulumi/pulumi-alicloud` repo](https://github.com/pulumi/pulumi-alicloud/issues); however, if that doesn't turn up anything,
+> please consult the source [`terraform-providers/terraform-provider-alicloud` repo](https://github.com/terraform-providers/terraform-provider-alicloud/issues).
+<h3>Modules</h3>
```

Comment:
Looks like an empty line needs to be added to the template in order to create some space between the `>` and the `<h3`>.

Reply:
This is fixed with the latest set of commits.